.gitignore

```diff
@@ -1,20 +1 @@
-/_output
```

Comment:
pretty sure we want this and the IDE ones.

Reply:
`_output` is not generated anymore

IDE ones don't belong there but to user's personal global .gitignore file

Reply:
> IDE ones don't belong there but to user's personal global .gitignore file

Perhaps purity-wise, but I don't see a reason to make life difficult.  Also, just saw that you updated this.  If it goes green then I'll close mine.